File: epfl-hc/hc2012/graders/TakeawayLib3.java
/////////////////////////////
// DO NOT SUBMIT THIS FILE //
/////////////////////////////

public class TakeawayLib3 {
  public static void main(String args[]) {
     Takeaway3 t3 = new Takeaway3();
          if (t3.ready_for_contest().equals("We hereby digitally sign the hc2 terms and conditions")) {
                  System.out.printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
                  return ;
          }
          if (t3.ready_for_contest().equals("Hiermit unterzeichnen wir digital die hc2 Bedingungen und Konditionen")) {
                  System.out.printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
                  return ;
          }
          if (t3.ready_for_contest().equals("Par la presente, nous signons numeriquement les conditions de participation au concours hc2")) {
                  System.out.printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
                  return ;
          }
          System.out.printf("Incorrect\n");
          return ;
  }
}

File: epfl-hc/hc2012/graders/takeawayLib3.c
/////////////////////////////
// DO NOT SUBMIT THIS FILE //
/////////////////////////////

#include <stdlib.h>
#include <stdio.h>
#include <string.h>
#include "takeawayLib3.h"

int main() {
	if (!strcmp(ready_for_contest(), "We hereby digitally sign the hc2 terms and conditions")) {
		printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
		return 0;
	}
	if (!strcmp(ready_for_contest(), "Hiermit unterzeichnen wir digital die hc2 Bedingungen und Konditionen")) {
		printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
		return 0;
	}
	if (!strcmp(ready_for_contest(), "Par la presente, nous signons numeriquement les conditions de participation au concours hc2")) {
		printf("51785494e5ad0efbb1856b0a4a9b8422Correct\n");
		return 0;
	}
	printf("Incorrect\n");
	return 0;
}
